processing/ip_handler_test.go

```diff
@@ -40,6 +40,9 @@ func makeIPHTTPEvent(srcip string, dstip string) types.Entry {
 		HTTPMethod: "GET",
 	}
 	eve := types.EveEvent{
+		Timestamp: &types.SuriTime{
+			Time: time.Now(),
```

Comment:
Yes, might not be important, but I rather ask anyway as this is all going to some kind of data store :)
Not `UTCNow()`?

Reply:
None of this is going to a data store. It's just generating mock test data that would come from an outside input in a real case.